packages/github/src/github.ts

```diff
@@ -14,7 +14,12 @@ export const context = new Context.Context()
  */
 export function getOctokit(
   token: string,
-  options?: OctokitOptions
+  options?: OctokitOptions,
+  ...additionalPlugins: OctokitPlugin[]
 ): InstanceType<typeof GitHub> {
+  if (additionalPlugins.length) {
```

Comment:
Do you need to check if `additionalPlugins` has any values?

If additionalPlugins is empty, then it seems like an empty list[1] will be appended to the defaults[2].

1: https://github.com/octokit/core.js/blob/ebd1baf5fc5bb9652504990c669f358630221381/src/index.ts#L64

2: https://github.com/actions/toolkit/blob/64c334f0e54c45f0ee319b5707281d9df0c43a3d/packages/github/src/utils.ts#L20

Reply:
oh so you're saying I can just skip the check altogether?

Reply:
Not a TS expert, but the following looks like it will be functionally the same as checking `additionalPlugins` for length.

```
{
  const GitHub = GitHub.plugin(...additionalPlugins);
  return new GitHub(getOctokitOptions(token, options));
}
```

Reply:
yeah that's a fair point